README.md

```diff
@@ -284,6 +159,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You have a LICENCE.md file, not just LICENSE, kindly add the extension for it to work, at the moment, it is not yet working.
![image](https://user-images.githubusercontent.com/86200356/234558397-46946f50-a0fb-41de-a6c5-df6379d5a621.png)

Reply:
@Olamarx Thank you for the feedback.